.circleci/config.yml

```diff
@@ -5,23 +5,21 @@
 ## Job to run the update path and tests.
 build_and_deploy: &build_and_deploy
   docker:
-    - image: cityofportland/drupal-ci:latest
+    - image: cimg/php:8.0-node
       auth:
         username: egovpdx
         password: $DOCKERHUB_PASSWORD
-  working_directory: /var/www/html
+  working_directory: /home/circleci/app
   steps:
     - restore_cache:
         name: Retore Git cache before checkout
         keys:
-          - source-v1-{{ .Branch }}-{{ .Revision }}
-          - source-v1-{{ .Branch }}-
```

Comment:
Why did you remove the fallback keys?

Reply:
Their documentation examples include fallback keys just like these so I'm curious why you removed them. https://circleci.com/docs/2.0/caching/